src/app/components/aboutPage/aboutPage.component.tsx

```diff
@@ -1,10 +1,103 @@
 import './aboutPage.component.scss';
+import rocket from '../../images/rocket.svg';
 
 export function AboutPage() {
     return (
-        <div className="about-page-component">
+        <div className="about-page-items">
             <h2>About page</h2>
-            'about page content - change me'
+
+            <div className="about-page-content">
+                <div className="about-page-header">
+                    <img alt="rocket" src={rocket} />
+                    <h3>Some facts</h3>
+                </div>
+                <div className="about-page-container">
+                    <ul>
+                        <li>There are over 140 spaceflight launches in 2021</li>
+                        <li>
+                            There are at least 7 people on space station right
+                            now
+                        </li>
+                        <li>
+                            In 24 hours, the space station is travelling through
+                            16 sunrises and sunsets
+                        </li>
+                        <li>
+                            The space for living and working on space station is
+                            larger than a six-bedroom house
+                        </li>
+                    </ul>
+                    <p>
+                        <em>
+                            Source:
+                            <br />
+                            <a
+                                href="https://www.nasa.gov/feature/facts-and-figures"
+                                target="_blank"
+                            >
+                                International Space Station Facts and Figures
+                            </a>
+                        </em>
+                    </p>
+                </div>
+            </div>
+
+            <div className="about-page-content">
+                <div className="about-page-header">
+                    <img alt="rocket" src={rocket} />
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt "Think of it like this: When choosing alt strings for your images, imagine what you would say when reading the page to someone over the phone without mentioning that there's an image on the page."

alt tag is optional but if you use it please use it properly

Reply:
All alts corrected.